packages/integrations/image/package.json

```diff
@@ -38,7 +38,7 @@
     "build": "astro-scripts build \"src/**/*.ts\" && tsc && pnpm run postbuild",
     "build:ci": "astro-scripts build \"src/**/*.ts\" && pnpm run postbuild",
     "postbuild": "astro-scripts copy \"src/**/*.wasm\"",
-    "dev": "astro-scripts dev \"src/**/*.ts\"",
+    "dev": "astro-scripts dev --copy-wasm \"src/**/*.ts\"",
```

Comment:
Isn't this just to copy [vendored wasm files](https://github.com/withastro/astro/blob/694918a56b01104831296be0c25456135a63c784/scripts/cmd/build.js#L114) of the core assets feature?

Reply:
Yes you're right, this does nothing here. I'll remove it.